sdk/python/feast/feature_store.py

```diff
@@ -693,99 +694,6 @@ def _get_feature_views_to_materialize(
 
         return feature_views_to_materialize
 
-    def plan(
```

Comment:
any reason for changing the position for this function

Reply:
By mistake